src/modules/claim/resolvers/revoker.resolver.ts

```diff
@@ -1,7 +1,9 @@
 import { IRevokerDefinition } from '@energyweb/credential-governance';
 import { RevokerResolver } from '@energyweb/vc-verification';
+import { Injectable } from '@nestjs/common';
 import { RoleService } from '../../role/role.service';
 
+@Injectable()
```

Comment:
@whitneypurdum I wonder if this might help fix the bug you've been looking into 👀